website/source/docs/agent/services.html.md

```diff
@@ -151,6 +155,17 @@ are used to configure the proxy and are specified in the [proxy
 docs](/docs/connect/proxies.html). If `native` is true, it is an error to also
 specifiy a managed proxy instance.
 
+The `weights` field is an optional field to specify the weight of a service in
+DNS SRV responses. If this field is not specified, its default value is:
+`"weights": {"passing": 1, "warning": 1}`.
+When a service is `critical`, it is excluded from DNS responses, `warning` services
+will be in DNS responses as well unless the `only_passing = true` is set in the
```

Comment:
nit: this reads a little confusingly and might be interpreted backwards, instead we could simplify it into multiple sentences:

> When a service is `critical`, it is excluded from DNS responses. Services with `warning` checks are in _included_ in responses by default, but excluded if the optional param `only_passing = true` is present in agent DNS configuration or `?passing` is used via the API.

Reply:
DONE